Project.toml

```diff
@@ -6,7 +6,9 @@ version = "0.25.15"
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+LambertW = "984bce1d-4616-540c-a9ee-88d1112d94c9"
```

Comment:
Maybe it would make sense to define this distribution in StatsFuns rather than using a separate package?

@jlapeyre Would you be OK with that?

Reply:
IIRC there was a discussion about moving it to SpecialFunctions and there even exists a PR.

Reply:
Ah yes that's https://github.com/JuliaMath/SpecialFunctions.jl/pull/84. Though it's quite outdated now and there have been new commits in LambertW since then.

@emfeltham Do you feel like reviving this PR (or opening a new one)? There seems to be lots of interest in it.